statistics/handle/update_test.go

```diff
@@ -1067,7 +1067,7 @@ func TestQueryFeedbackForPartition(t *testing.T) {
 		{
 			// test primary key feedback
 			sql: "select * from t where t.a <= 5",
-			hist: "column:1 ndv:2 totColSize:2\n" +
+			hist: "column:1 ndv:2 totColSize:0\n" +
```

Comment:
After analyze, `totColSize` is `0`. Then it becomes `2` after updating feedback. However, when I dump stats delta before analyze, `totColSize` is always `0`. I'm not very clear why and still investigating it...